src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -5,44 +5,70 @@
 import java.util.List;
 
 import seedu.address.commons.core.index.Index;
+import seedu.address.commons.core.index.IndexList;
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes 1 or more people identified using it's displayed index from the address book.
  */
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes 1 or more people identified by their index numbers used in the displayed person list.\n"
```

Comment:
From the description alone, it isn't clear how to use the command for multiple people. (comma separated, space separated, etc.?)